core-bundle/assets/scripts/core.js

```diff
@@ -1502,6 +1506,7 @@ window.Backend =
 								ntr.inject(tr, 'after');
 								addEventsTo(ntr);
 								makeSortable(tbody);
+                                ntr.getFirst('td').getFirst('input').select();
```

Comment:
```suggestion
								ntr.getFirst('td').getFirst('input').select();
```

Reply:
Fixed in 6031cc5e604cf747b073b7ff7b763b3d7c2f99d5.